### PR TITLE
Relax manual analysis criteria to only check chosenVariationId

### PR DESCRIPTION
**Pull request by aaronyan**
While looking at some of our experiment results, I kept getting frustrated with the manual analysis required. In with hopefully a quick fix! :)

## How has this been tested?

- [x] Manual testing with production data (locally)

### CODE REVIEW

File: src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ export default function CondensedLatestAnalyses({
         .map(
           (analyses) =>
             //  istanbul ignore next; We don't need to test empty analyses as we filter out all undefined values
-            _.last(analyses)?.recommendation,
+            _.last(analyses)?.recommendation.chosenVariationId,
         )
         .filter((recommendation) => !!recommendation)
       const uniqueRecommendations = _.uniq(recommendations.map((recommendation) => JSON.stringify(recommendation)))
```
Comment (aaronyan) [inline src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx:91]: Should we change the name throughout this file from `recommendations ` to `recommendedVariationIds`?

Comment (jessie-ross) [reply src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx:91]: Yep, I think this is needed to make this code clear.

We can also remove the `JSON.stringify` mapping.

Comment (yanirs) [inline src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx:91]: In addition to the naming, it looks like the `filter()` call won't work as expected. How about we change it to something like this?

```suggestion
            _.last(analyses)?.recommendation,
        )
        .filter((recommendation) => !!recommendation)
      const recommendationConflict = _.uniq(recommendations.map((recommendation) => recommendation.chosenVariationId)) > 1
```

That is, leave the `recommendations` array as it is, and change the `recommendationConflict` variable.

Comment (jessie-ross) [reply src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx:91]: I don't see why filter won't work?
Also before this PR goes out `chosenVariationId` will be optional chained, in case that is something you are missing?

Comment (jessie-ross) [reply src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx:91]: Ahh I didn't realise `chosenVariationId` set to `null` signified something.

**Comment by aaronyan** [comment]
Changed to 'WIP' and I'll save this for next week to avoid further distraction. :) Thanks for the eyes folks!

**Comment by jessie-ross** [comment]
This was so close to being done I thought I might just gently nudge it across the line 😄

**Review by yanirs — APPROVED** [review]
Pushed https://github.com/Automattic/abacus/pull/405/commits/e58581b6a858fb6eacc4be0fa51d8311d1ec533d to make it a true team effort. :smile: 

@jessie-ross you can merge if it looks good to you.

**Review by jessie-ross — APPROVED** [review]

**Comment by aaronyan** [comment]
Haha thank you @jessie-ross and @yanirs for helping make this happen! 😄
